loopd/config.go

```diff
@@ -183,6 +185,9 @@ type Config struct {
 	TotalPaymentTimeout time.Duration `long:"totalpaymenttimeout" description:"The timeout to use for off-chain payments."`
 	MaxPaymentRetries   int           `long:"maxpaymentretries" description:"The maximum number of times an off-chain payment may be retried."`
 
+	MaxStaticAddrHtlcFeePercentage       float64 `long:"maxstaticaddrhtlcfeepercentage" description:"The maximum fee percentage that the server can charge for the htlc tx."`
```

Comment:
In the worst case there are two transactions: HTLC and its sweep. Does this parameter cover only the former or the sum of both of them?

Reply:
This covers only the fees for the htlc tx as mentioned in the end of the comment.